packages/card/src/Card.ts

```diff
@@ -266,77 +272,105 @@ export class Card extends LikeAnchor(
+                              class="checkbox-toggle"
+                              @pointerdown=${this.stopPropagationOnHref}
+                          >
+                              <sp-checkbox
+                                  class="checkbox"
+                                  @change=${this.handleSelectedChange}
+                                  .checked=${this.selected}
+                              >
+                                  <span class="sr-only">
+                                      ${this.label || this.heading}
+                                  </span>
+                              </sp-checkbox>
+                          </sp-popover>
+                      `
+                    : nothing}
+                ${this.variant === 'quiet' && this.size === 's'
+                    ? html`
+                          <div
+                              class="spectrum-QuickActions actions"
+                              @pointerdown=${this.stopPropagationOnHref}
+                          >
+                              <slot name="actions"></slot>
                           </div>
                       `
                     : nothing}
             </div>
-            ${this.href
-                ? this.renderAnchor({
-                      id: 'like-anchor',
-                      labelledby: 'heading',
-                  })
-                : nothing}
-            ${this.variant === 'standard'
-                ? html`
-                      <slot name="footer"></slot>
-                  `
-                : nothing}
-            ${this.toggles
-                ? html`
-                      <sp-popover
-                          class="checkbox-toggle"
-                          @pointerdown=${this.stopPropagationOnHref}
-                      >
-                          <sp-checkbox
-                              class="checkbox"
-                              @change=${this.handleSelectedChange}
-                              ?checked=${this.selected}
-                              tabindex="-1"
-                          ></sp-checkbox>
-                      </sp-popover>
-                  `
-                : nothing}
-            ${this.variant === 'quiet' && this.size === 's'
-                ? html`
-                      <div
-                          class="spectrum-QuickActions actions"
-                          @pointerdown=${this.stopPropagationOnHref}
-                      >
-                          <slot name="actions"></slot>
-                      </div>
-                  `
-                : nothing}
         `;
     }
 
     protected override firstUpdated(changes: PropertyValues): void {
         super.firstUpdated(changes);
+        if (changes.has('label')) {
+            if (this.label) {
+                this.setAttribute('aria-label', this.label);
+            } else {
+                this.removeAttribute('aria-label');
+            }
+        }
```

Comment:
this is redundant since update will be called during the initial update as well. We can remove this logic from firstUpdated and keep it only in update.